clients/cli/src/main/java/org/apache/gravitino/cli/CommandContext.java

```diff
@@ -55,6 +56,8 @@ public CommandContext(CommandLine line) {
             ? line.getOptionValue(GravitinoOptions.OUTPUT)
             : Command.OUTPUT_FORMAT_PLAIN;
     this.quiet = line.hasOption(GravitinoOptions.QUIET);
+    // TODO add limit option to CLI
+    this.outputLimit = -1;
```

Comment:
Pleas rethink this design...

If we have received a long list to show in the table, and the purpose of this option is to show only part of it, that would be a bad design.
A more common pattern for LIST is to provide `start` and `limit` as a pair. These two options can be used to control how many items you want to retrieve from the server, not how many items you want to show in a tabular format. These two options can also be used to support paging, i.e. retrieve things in trunks.

My previous feedback wasn't that `limit` is useless. It is specific to LIST operations. You won't use it when you CREATE/UPDATE/DELETE/PATCH something.

Reply:
Let's drop the output limit for now; that can be sorted in another PR. Plus, I'm really not sure that it's needed right now.